### PR TITLE
Generalized pricing model to use non 365 day terms

### PR DESCRIPTION
**Pull request by jalextowle**

### CODE REVIEW

**Review by jrhea — COMMENTED** [review]
the math looks correct to me, but i don't know how this will impact the rest of the sim

**Review by sentilesdal — APPROVED** [review]
lgtm

File: config/example_config.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ floor_fee = 0  # minimum fee percentage (bps)
 # fixed variables
 num_trading_days = 180  # should be <= pool_duration
 num_blocks_per_day = 7200 # 24 * 60 * 60 / 12 = 12 second block time
-token_duration = 0.24657534246575342465753424657534  # 90 / 365 = time lapse between token mint and expiry as a fraction of a year
+num_position_days = # 90 = time lapse between token mint and expiry as days
```
Comment (dpaiton) [inline config/example_config.toml:27]: I think this has a typo with the location of `#`?

**Review by dpaiton — APPROVED** [review]
lgtmt

**Pull request merged**